app/templates/crate/versions.hbs

```diff
@@ -16,22 +16,27 @@
   </div>
 </div>
 
-<ul local-class="list">
+<ul local-class="list" class="{{unless this.sortedVersions 'is-empty'}}">
```

Comment:
what is the purpose of the `'is-empty'` here?

Reply:
I use this to determine if it's the initial load. Therefore, when there are empty versions (indicating loading is still in progress), it provides a margin and displays a different loading style (text style) compared to when versions are already present (button style).

Reply:
ohhhh, that is an inline `unless`. nevermind 😅